src/mapping/query/function/format_timestamp.rs

```diff
@@ -18,20 +18,21 @@ impl FormatTimestampFn {
 }
 
 impl Function for FormatTimestampFn {
-    fn execute(&self, ctx: &Event) -> Result<Value> {
-        let format = match self.format.execute(ctx)? {
+    fn execute(&self, ctx: &Event) -> Result<QueryValue> {
+        let format = match self.format.execute(ctx)?.into() {
```

Comment:
While we're updating these, I suggest converting them to `required!` or `optional!`, as that will reduce all the `into` calls we're doing right now?

Although we could also defer that to a follow-up PR.